_maps/map_files/Pahrump-Sunset/RockSprings.dmm

```diff
@@ -20846,8 +20856,8 @@
 "klx" = (
 /obj/structure/closet/crate/grave,
 /obj/item/clothing/accessory/necklace{
-	name = "Raspberry Locket";
-	desc = "That's a locket with a charm that looks like a raspberry.  It ssmells nice for something that has been burried for a while with a corpse."
+	desc = "That's a locket with a charm that looks like a raspberry.  It ssmells nice for something that has been burried for a while with a corpse.";
```

Comment:
misspelling. ssmells and burried. intentional?

Reply:
dunno, didn't add it. gonna fix it anyway